src/useBaseQuery.js

```diff
@@ -89,6 +89,7 @@ export function useBaseQuery(queryKey, queryVariables, queryFn, config = {}) {
 
   // Handle refetch interval
   React.useEffect(() => {
+    const query = queryRef.current
     if (
       config.refetchInterval &&
       (!query.refetchInterval || config.refetchInterval < query.refetchInterval)
```

Comment:
I think `query.refetchInterval` is confusing and should be renamed to `query.refetchIntervalId`, since it actually stores interval ID.

Then `config.refetchInterval < query.refetchInterval` comparison doesn't make sense to me.

As well as `!query.refetchInterval` check, since `query.refetchInterval` will be always `undefined` due to cleaning function returned by useEffect.

@tannerlinsley thoughts?

Reply:
Yeah, the idea was that if a user passed different intervals for the same query but on different instances, then the shortest interval time would override the longer one.

Reply:
Should work as you described now!